<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">

    
    <script type="text/javascript" src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML,/mathjax_local"></script>
    
    <title><h2>Tammo Rukat</h2></title>

    <!-- Bootstrap Core CSS -->
    <link href="/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom CSS -->
    <link href="/css/business-casual.css" rel="stylesheet">

    <!-- Fonts -->
    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Josefin+Slab:100,300,400,600,700,100italic,300italic,400italic,600italic,700italic" rel="stylesheet" type="text/css">

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
        <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

</head>

<body>
    <div class="brand">Tammo Rukat</div>
    <div class="address-bar"> Statistics - Machine Learning - Genomics | DPhil Student @ University of Oxford </div>
    <!-- Navigation -->
    <nav class="navbar navbar-default" role="navigation">
        <div class="container">
            <!-- Brand and toggle get grouped for better mobile display -->
            <div class="navbar-header">
                <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                    <span class="sr-only">Toggle navigation</span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </button>
                <!-- navbar-brand is hidden on larger screens, but visible when the menu is collapsed -->
                <a class="navbar-brand" href="index.html">Business Casual</a>
            </div>
            <!-- Collect the nav links, forms, and other content for toggling -->
            <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                <ul class="nav navbar-nav">
                    <li>
                        <a href="/index">Blog</a>
                    </li>
		    <li>
                        <a href="/publications">Publications</a>
                    </li>
                    <li>
                        <a href="/about">About</a>
                    </li>
                </ul>
            </div>
            <!-- /.navbar-collapse -->
        </div>
        <!-- /.container -->
    </nav>

        <div class="container">
            <div class="row">
              <div class="box">
		<h1 id="journal-papers"><small>journal papers</small></h1>
<ul>
  <li>
    <p><a href="https://arxiv.org/abs/1702.06166"><strong>Bayesian Boolean Matrix Factorisation</strong></a><br /> T. Rukat, C.C. Holmes, M. Titsias, C. Yau. ICML, 2017. preprint arXiv:1702.06166</p>

    <p><button type="button" class="btn btn-default" data-toggle="collapse" data-target="#boomf-abstract">abstract</button>
  <button type="button" class="btn btn-default" data-toggle="collapse" data-target="#boomf-bibtex">bibtex</button></p>

    <div id="boomf-abstract" class="collapse">
  Boolean matrix factorisation aims to decompose a binary data matrix into an approximate Boolean product of two low rank, binary matrices: one containing meaningful patterns, the other quantifying how the observations can be expressed as a combination of these patterns. We introduce the OrMachine, a probabilistic generative model for Boolean matrix factorisation and derive a Metropolised Gibbs sampler that facilitates efficient parallel posterior inference. On real world and simulated data, our method outperforms all currently existing approaches for Boolean matrix factorisation and completion. This is the first method to provide full posterior inference for Boolean Matrix factorisation which is relevant in applications, e.g. for controlling false positive rates in collaborative filtering and, crucially, improves the interpretability of the inferred patterns. The proposed algorithm scales to large datasets as we demonstrate by analysing single cell gene expression data in 1.3 million mouse brain cells across 11 thousand genes on commodity hardware.
  </div>
    <div id="boomf-bibtex" class="collapse">
  <pre>
  @article{rukat2017_bayes-boolean_icml_pre,
  title = {Bayesian Boolean Matrix Factorisation},
  author = {Rukat, Tammo and Holmes, Chris C. and Titsias, Michalis K. and Yau, Christopher},
  journal = {Proceedings of the 34th Annual International Conference on Machine Learning; preprint arXiv:1702.06166},
  year = 2017}</pre>
  </div>
  </li>
  <li>
    <p><a href="http://onlinelibrary.wiley.com/doi/10.1002/mrm.25319/abstract"><strong>Dynamic contrast-enhanced MRI in mice: an investigation of model parameter uncertainties.</strong></a><br />
T. Rukat, S. Walker-Samuel, and S. A. Reinsberg. Magnetic Resonance in Medicine, 2014.</p>

    <p><button type="button" class="btn btn-default" data-toggle="collapse" data-target="#rukat2015-abstract">abstract</button>
  <button type="button" class="btn btn-default" data-toggle="collapse" data-target="#rukat2015-bibtex">bibtex</button></p>
    <div id="rukat2015-abstract" class="collapse">
  PURPOSE:<br />
  To establish the experimental factors that dominate the uncertainty of hemodynamic parameters in commonly used pharmacokinetic models.<br />
  METHODS:<br />
  By fitting simulation results from a multiregion tissue exchange model (Multiple path, Multiple tracer, Indicator Dilution, 4 region), the precision and accuracy of hemodynamic parameters in dynamic contrast-enhanced MRI with four tracer kinetic models is investigated. The impact of various injection rates as well as imprecise knowledge of the arterial input functions is examined.<br />
  RESULTS:<br />
  Fast injections are beneficial for K(trans) precision within the extended Tofts model and within the two-compartment exchange model but do not affect the other models under investigation. Biases from errors in the arterial input functions are mostly consistent in size and direction for the simple and the extended Tofts model, while they are hardly predictable for the other models. Errors in the hematocrit introduce the greatest loss in parameter accuracy, amounting to an average K(trans) bias of 40% for a 30% overestimation throughout all models.<br />
  CONCLUSION:<br />
  This simulation study allows the detailed inspection of the isolated impact from various experimental conditions on parameter uncertainty. Because parameter uncertainty comparable to human studies was found, this study represents a validation of preclinical dynamic contrast-enhanced MRI for modeling human tumor physiology.
  </div>
    <div id="rukat2015-bibtex" class="collapse">
  <pre>@article{rukat2015_dynam-mri,
author =	 {Rukat, T. and Walker-Samuel, S and Reinsberg, S. A.},
doi =		 {10.1002/mrm.25319},
issn =	 {1522-2594},
journal =	 {Magnetic Resonance in Medicine},
pages =	 {1979-1987},
title =	 {Dynamic Contrast-Enhanced {MRI} in Mice: An Investigation of Model Parameter Uncertainties},
volume =	 73,
year =	 2015}</pre>
  </div>
  </li>
  <li>
    <p><a href="http://aip.scitation.org/doi/10.1063/1.4719530"><strong>Chain-length dependent growth dynamics of n-alkanes on silica investigated by<br /> energy-dispersive x-ray reflectivity in situ and in real-time.</strong></a><br />
C. Weber, C. Frank, S. Bommel, T. Rukat, W. Leitenberger, P. Schäfer, F. Schreiber, and S. Kowarik.<br /> The Journal of Chemical Physics. 2012.</p>

    <p><button type="button" class="btn btn-default" data-toggle="collapse" data-target="#weber2012-chain-abstract">abstract</button>
  <button type="button" class="btn btn-default" data-toggle="collapse" data-target="#weber2012-chain-bibtex">bibtex</button></p>
    <div id="weber2012-chain-abstract" class="collapse">
  We compare the growth dynamics of the three n-alkanes C(36)H(74), C(40)H(82), and C(44)H(90) on SiO(2) using real-time and in situ energy-dispersive x-ray reflectivity. All molecules investigated align in an upright-standing orientation on the substrate and exhibit a transition from layer-by-layer growth to island growth after about 4 monolayers under the conditions employed. Simultaneous fits of the reflected intensity at five distinct points in reciprocal space show that films formed by longer n-alkanes roughen faster during growth. This behavior can be explained by a chain-length dependent height of the Ehrlich-Schwoebel barrier. Further x-ray diffraction measurements after growth indicate that films consisting of longer n-alkanes also incorporate more lying-down molecules in the top region. While the results reveal behavior typical for chain-like molecules, the findings can also be useful for the optimization of organic field effect transistors where smooth interlayers of n-alkanes without coexistence of two or more molecular orientations are required.
  </div>
    <div id="weber2012-chain-bibtex" class="collapse">
  <pre>@article{weber2012_chain,
  author =	 {Weber, C. and Frank, C. and Bommel, S. and Rukat,
  T. and Leitenberger, W. and Sch{\"a}fer, P. and
  Schreiber, F. and Kowarik, S.},
  doi =		 {10.1063/1.4719530},
  issn =	 {1089-7690},
  journal =	 {The Journal of chemical physics},
  number =	 20,
  pages =	 204709,
  title =	 {Chain-Length Dependent Growth Dynamics of N-Alkanes
  on Silica Investigated By Energy-Dispersive X-Ray
  Reflectivity in Situ and in Real-Time.},
  url =		 {http://www.ncbi.nlm.nih.gov/pubmed/22667583},
  volume =	 136,
  year = 2012,}</pre>
  </div>
  </li>
</ul>

<h1 id="-conference-posters-and-workshops-"><small> conference posters and workshops </small></h1>
<ul>
  <li>
    <p><a href="https://arxiv.org/abs/1606.02344"><strong>Resting state brain networks from EEG: Hidden Markov states vs classical microstates</strong></a><br />
T. Rukat, A. Baker, A. Quinn, M. Woolrich. MLINI workshop at NIPS 2015</p>

    <p><a href="https://arxiv.org/abs/1606.02344" class="btn btn-default" role="button">article</a>
  <a href="/content/posters/nips_poster_2015.pdf" class="btn btn-default" role="button">poster</a>
<!-- <a href="/content/bibtex/Rukat2015a.txt" target="_blank">[bibtex]</a>  --></p>
  </li>
  <li>
    <p><a href="/content/posters/dce_weighting_2015.pdf"><strong>Fully Bayesian Multi-Model Inference for Parameter Estimation in DCE-MRI</strong></a><br />
T. Rukat and S. A. Reinsberg.  Proc. Intl. Soc. Mag. Reson. Med. 23 (2015) 2339, 2015.<br />
<!-- <a href="/content/bibtex/Rukat2015a.txt" target="_blank">[bibtex]</a>  --></p>
  </li>
  <li>
    <p><a href="/content/posters/Poster2014.pdf"><strong>Information Criteria weighted Parameter Estimates in DCE-MRI.</strong></a><br />
T. Rukat and S. A. Reinsberg.  Proc. Intl. Soc. Mag. Reson. Med. 22 (2014) 2741, 2014. 
 <!-- <a href="/content/bibtex/Rukat2014.txt" target="_blank">[bibtex]</a> --></p>
  </li>
  <li>
    <p><a href="/content/posters/Poster2013.pdf"><strong>AIF Induced Limits of Parameter Uncertainty in Pharmacokinetic Models of Pre-Clinical DCE-MRI.</strong></a><br />
T. Rukat, S. Walker-Samuel, and S. A. Reinsberg. Proc. Intl. Soc. Mag. Reson. Med. 21 (2013) 2214, 2013. 
 <!-- <a href="/content/bibtex/Rukat2013.txt" target="_blank">[bibtex]</a> --></p>
  </li>
</ul>

<h1 id="-theses-"><small> theses </small></h1>
<ul>
  <li>
    <p><a href="/content/theses/Master.pdf"><strong>Parameter Uncertainties in Tracer Kinetic Modelling of Dynamic Contrast Enhanced MRI.</strong></a><br />
Master Thesis, Humboldt Universität zu Berlin / University of British Columbia, October 2013.
<!-- <a href="/content/bibtex/Rukat2013a.txt" target="_blank">[bibtex]</a> --></p>
  </li>
  <li>
    <p><a href="/content/theses/Bachelor.pdf"><strong>Modeling Organic Molecule Thin Film Growth – 
From In Situ X-Ray Reflectivity to Atomistic Growth Processes.</strong></a><br />
Bachelor Thesis, Humboldt Universität zu Berlin, September 2011.</p>
  </li>
</ul>

<h1 id="-others-"><small> others </small></h1>
<ul>
  <li>
    <p><a href="/content/misc/Tammo_Rukat_Project_1.pdf"><strong>Distributed analysis of expression quantitative trait loci in Apache Spark</strong></a><br />
T. Rukat, University of Oxford. Short Project Report within the <a href="http://www.sabsidc.ox.ac.uk/">SABS CDT</a> doctoral programme.
<!-- <a href="/content/bibtex/Rukat_Spark_eQTL.txt" target="_blank">[bibtex]</a>  --></p>
  </li>
  <li>
    <p><a href="/content/misc/Tammo_Rukat_Project_2.pdf"><strong>Identification of resting state brain networks from EEG with simultaneous MEG</strong></a><br />
T. Rukat, University of Oxford. Short Project Report within the <a href="http://www.sabsidc.ox.ac.uk/">SABS CDT</a> doctoral programme.
<!-- <a href="/content/bibtex/Rukat_Spark_eQTL.txt" target="_blank">[bibtex]</a>  --></p>
  </li>
</ul>

	      </div>
	   </div>
	</div>
	
    <!-- /.container -->

    <!-- <footer> -->
    <!--     <div class="container"> -->
    <!--         <div class="row"> -->
    <!--             <div class="col-lg-12 text-center"> -->
    <!--                 <p>Copyright &copy; Your Website 2014</p> -->
    <!--             </div> -->
    <!--         </div> -->
    <!--     </div> -->
    <!-- </footer> -->

    <!-- jQuery -->
    <script src="/js/jquery.js"></script>

    <!-- Bootstrap Core JavaScript -->
    <script src="/js/bootstrap.min.js"></script>

    <!-- Script to Activate the Carousel -->
    <script>
    $('.carousel').carousel({
        interval: 5000 //changes the speed
    })
    </script>

</body>

</html>
